packages/qwik-react/src/react/types.ts

```diff
@@ -14,6 +14,7 @@ export interface QwikifyBase {
    * **Use case:** Immediately-visible UI elements that need to be interactive as soon as possible.
    */
   'client:load'?: boolean;
+  'qwik-client:load'?: boolean;
```

Comment:
why `qwik-client` Why no just `qwik`? 🤷‍♂️

Reply:
I thought it would be less clear that it is a hydration directive, but after getting some feedback changing it to just `qwik`